src/Umbraco.Web.UI.Client/src/common/directives/components/content/edit.controller.js

```diff
@@ -941,15 +941,13 @@
             const openPreviewWindow = () => {
                 // Chromes popup blocker will kick in if a window is opened
                 // without the initial scoped request. This trick will fix that.
-                //
-                const previewWindow = $window.open('preview/?init=true', 'umbpreview');
+              
+              const previewWindow = $window.open(`preview/?id=${content.id}${$scope.culture ? `&culture=${$scope.culture}` : ''}`, 'umbpreview');
+
+              previewWindow.addEventListener('load', () => {
+                previewWindow.location.href = previewWindow.document.URL;
+              });
 
-                // Build the correct path so both /#/ and #/ work.
```

Comment:
Just a not for future reference: I have been searching what this means with `/#/` and `#/` - I think it was because the preview URL probably used to be built from part of the current backoffice URL (like `https://localhost:44331/umbraco#/content/content/edit/1666?cculture=nl-NL&mculture=en-US`) and that changed over time, but everyone just kept this code around because it "seemed" important. From testing now, it seems completely unnecessary indeed! 👍